neural_speed/core/layers/mha_dense.cpp

```diff
@@ -72,7 +72,9 @@ bool bestla_reordered_attn_fp32_support(const attn_shape_t* params) {
   // TODO(Yi): check K V's layout
   if (_cd->AMX_BF16()) return true;
 #endif
-  return !_cd->AVX512F() || _cd->AVX2();  // use avx2 and f16c on avx2 platforms
+  // use avx2 and f16c on avx2 platforms
+  // todo: check avx2 mha on sever
+  return !_cd->AVX512F() && _cd->AVX2();
```

Comment:
what does this mean? no avx512f but has avx2. Does it mean that MHA_AVX512F is not ready?

Reply:
We have not MHA_AVX512F according to @DDEle. And MHA_AVX2 is wrong on sever.

Reply:
Are you sure that AVX2 has no problem on client as it's wrong on server?

Reply:
#137 test llama on client and seems no problem.